### PR TITLE
Revise header.html for easier readability for future updates

### PR DESCRIPTION
**Pull request by kenman345**
Saw this and thought it was a bit hard to read for no good reason.

Also, I revised the bottom section in the same way as the top even though it never is shown/rendered right now. Didnt know enough to determine if we should put it back in or remove the code block entirely.

### CODE REVIEW

**Review by fpigerre — DISMISSED** [review]

**Comment by kenman345** [comment]
I know this is approved, and the top of the page has value but in another PR we mentioned removing dead code. Can we get this merged so I can remove the bottom half out cleanly?

File: _includes/header.html [hunk 1]
```diff
@@ -1,12 +1,8 @@
 <div class="ui teal inverted menu">
-  {% if include.page == 'home' %}
-  <a href="/" class="active item">
-    {% else %}
-    <a href="/" class="item">
-      {% endif %}
-      <i class="home icon"></i>
-      Home
-    </a>
+  <a href="/" class="{% if include.page == 'home' %}active {% endif %}item">
```
Comment (Carlgo11) [inline _includes/header.html]: Is this even used?

File: _includes/header.html [hunk 2]
```diff
@@ -1,12 +1,8 @@
...
+    <i class="home icon"></i>
+    Home
+  </a>
 
     {% if include.page == 'home' %}
```
Comment (Carlgo11) [inline _includes/header.html]: If I'm not mistaken 'home' is the old main page so there's not really any reason for keeping this section.

File: _includes/header.html [hunk 3]
```diff
@@ -38,14 +34,10 @@
 
     {% if 1 == 0 %}
     <!-- Don't show for right now -->
-    {% if include.page == 'restrictions' %}
```
Comment (Carlgo11) [inline _includes/header.html:41]: There's no restrictions page so this part can also be removed.

**Review by Carlgo11 — COMMENTED** [review]
Maybe @psgs can confirm my comments before we remove anything.

Other than that everything looks good. 👍

File: _includes/header.html (hunk 1 above)
Comment (fpigerre) [reply _includes/header.html]: Good catch, I completely forgot we removed this :+1:

**Review by fpigerre — DISMISSED** [review]

**Comment by kenman345** [comment]
this was appproved by one person Jan 17th, what the hold up?

**Comment by Carlgo11** [comment]
https://github.com/2factorauth/twofactorauth/pull/3566/files#discussion_r247767107

File: _includes/header.html (hunk 2 above)
Comment (kenman345) [reply _includes/header.html]: not sure what part you are saying to remove. Wanna give some line numbers? havent looked at this in a while and github showing things a bit weird for me.

Comment (Carlgo11) [reply _includes/header.html]: L7 to L26.

**Comment by kenman345** [comment]
should be all set now!

**Review by Carlgo11 — APPROVED** [review]

**Comment by jtagcat** [comment]
Is this dead?

**Comment by kenman345** [comment]
Why dead? I revised this as recently as last week to make it in alignment
of what was asked. It’s ready to get a second approval and then merged

**Review by kmpoppe — APPROVED** [review]
Looks good

**Pull request merged**